src/classlibnative/bcltype/system.cpp

```diff
@@ -73,9 +73,11 @@ FCIMPLEND;
 
 
 
-#ifndef FEATURE_CORECLR
 INT64 QCALLTYPE SystemNative::GetWorkingSet()
 {
+#if PLATFORM_UNIX
+    return 0; // TODO: Implement in PAL
```

Comment:
> What if we were to introduce a "private" EnvironmentCore class, that's exposed publically from System.Private.Corelib.

I think this direction sounds interesting. We do have similar existing prior art in CoreRT: Internal.Runtime.Augments.

> The simplest way to reuse it would be reflection

I agree. There is a problem with it: How are folks going to discover that they need reference System.Diagnostics.Process to use Environment.WorkingSet? I know we have some use of it today and we have not done anything about it. It gets discovered by trial & error. We need to be careful to not cross a threshold where it becomes a real problem.